pkg/app/couchbase.go

```diff
@@ -52,6 +52,8 @@ type CouchbaseDB struct {
 }
 
 // Last tested woking version "2.3.0"
+//
+//nolint:stylecheck
```

Comment:
why did we have to add the `nolint` here?

Reply:
This is because style check expects exported function should have description of what its doing.
```
ST1020: comment on exported function NewCouchbaseDB should be of the form "NewCouchbaseDB ...
```

Reply:
I think instead of `nolint` we should just add `NewCouchbaseDB initialises an instance of Couchbase DB` or something in these line. We are anyway adding the comment of `last tested` and a new line after `last tested` doesn't make sense.

Reply:
done